bin/post_compile

```diff
@@ -23,7 +23,7 @@ fi
 echo "-----> Pushing source maps to Sentry"
 if [ -n "$SENTRY_API_KEY" ] && [ -n "$SENTRY_ORG" ] && [ -n "$SENTRY_PROJECT_NAME" ] && [ -n "$SOURCE_VERSION" ]; then
     npx @sentry/cli --auth-token=$SENTRY_API_KEY releases --org=$SENTRY_ORG --project=$SENTRY_PROJECT_NAME files $SOURCE_VERSION upload-sourcemaps ./frontend/webpack_bundles/ --url-prefix "~/static/webpack_bundles/" --rewrite
+    rm ./frontend/webpack_bundles/*.js.map
```

Comment:
Only delete the source map if it was previously created, avoiding a crash.

Reply:
Nice.